grafana-mimir.yaml

```
@@ -0,0 +1,43 @@
+package:
+  name: grafana-mimir
+  version: 2.12.0
+  epoch: 2
```

```suggestion
  epoch: 0
```

Since this is new the epoch should be 0